lizard_auth_client/forms.py

```diff
@@ -190,13 +190,14 @@ def clean(self):
             user_dict = client.sso_search_user_by_email(
                 self.cleaned_data['email'])
         except HTTPError as e:
+            error_text = e.response.text
             logger.info("Error when searching user on the SSO: %s",
-                        e.response.text)
+                        error_text)
```

Comment:
Error level logging komt in sentry terecht. En dit soort errors komen ook al in de SSO in sentry.

Hm. Misschien is het nog niet eens zo raar om het hier ook in sentry te laten komen...